bin/install-wp-tests.sh

```diff
@@ -151,6 +154,9 @@ install_db() {
 		fi
 	fi
 
+	# drop existing database
+	mysqladmin drop -f $DB_NAME --user="$DB_USER" --password="$DB_PASS"$EXTRA || echo "Database $DB_NAME does not exist"
```

Comment:
This command is a little noisy in the CI log because the database doesn't previously exist there.

```
+mysqladmin drop -f wc_admin_test --user=root --password= --host=localhost --protocol=tcp
Warning: Using a password on the command line interface can be insecure.
mysqladmin: DROP DATABASE wc_admin_test failed;
error: 'Can't drop database 'wc_admin_test'; database doesn't exist'
+echo 'Database wc_admin_test does not exist'
Database wc_admin_test does not exist
```

Perhaps it would be better to execute SQL directly so that `IF EXISTS` could be used? Something like:
```
DROP wc_admin_test IF EXISTS
```

Reply:
I've tried to use direct query but it did still produce an error message albeit shorter, so ended up with `2>/dev/null` to silence it out.